test/MOI_wrapper.jl

```diff
@@ -921,3 +921,29 @@ end
     @test MOI.get(model, MOI.ConstraintDual(), xl) == 1.0
     @test MOI.get(model, MOI.ConstraintDual(), xu) == 0.0
 end
+
+@testset "Objective functions" begin
+    model = Gurobi.Optimizer(GUROBI_ENV)
+    x = MOI.add_variable(model)
+    @test MOI.get(model, MOI.ObjectiveSense()) == MOI.FEASIBILITY_SENSE
+    @test MOI.get(model, MOI.ListOfModelAttributesSet()) == Any[MOI.ObjectiveSense()]
+    MOI.set(model, MOI.ObjectiveSense(), MOI.MIN_SENSE)
+    MOI.set(model, MOI.ObjectiveFunction{MOI.SingleVariable}(), MOI.SingleVariable(x))
```

Comment:
Support for `SingleVariable` should be dropped now that we have `Bridge.Objective.FunctionizeBridge`

Reply:
Agreed. But that can wait for a different PR.